README.md

```diff
@@ -111,14 +111,49 @@ The fastest way to get support is by [searching our documentation](https://acade
 
 ### Locally
 
-1. Start up an A3P instance:
+1. Start up an A3P instance
 
-```sh
-docker run -p 26657:26657 -p 1317:1317 -p 9090:9090 ghcr.io/agoric/agoric-3-proposals:latest
-```
+   ```sh
+   docker run -p 26657:26657 -p 1317:1317 -p 9090:9090 ghcr.io/agoric/agoric-3-proposals:latest
+   ```
+
```

Comment:
Why this step?
I see we have a3p as a separate service in docker compose and `yarn dev` should also spawn it.

<img width="625" alt="Screenshot 2024-12-24 at 7 15 35 PM" src="https://github.com/user-attachments/assets/b306a00a-e02a-44d0-9bac-0e51990beabe" />

Reply:
I was wondering why `ci` was necessary. Your comment here is helpful. Please include it somewhere in the README or source code.